src/Navigation.php

```diff
@@ -12,8 +12,7 @@
 /**
  * A simple container class for {@link Laminas\Navigation\Page} pages
  *
- * @template TPage of AbstractPage
- * @template-extends AbstractContainer<TPage>
+ * @template-extends AbstractContainer<AbstractPage>
  */
 class Navigation extends AbstractContainer
```

Comment:
If I get this correctly, with this change, `AbstractContainer` becomes extensible (accepting any kind of page), whereas `Navigation` only supports `AbstractPage`?

I suppose that's workable :thinking:

Reply:
Actually, now that you mention it, I don't think there's any value in `AbstractContainer` being generic either. But on the other hand, it isn't hurting anything. I don't know, what do you say?

Reply:
I'd say this is the right approach, it's only the genericness of the page as an issue as the generic type is used to specify child pages of a page, which could be anything that extends `AbstractPage`, it doesn't need to be restricted by a generic IMO.

Reply:
Fair. I'll put the `@template` back into `Navigation`.